PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/RootCamera/ClassicCamera.rbxmx

```diff
@@ -47,20 +47,11 @@ end
 local function CreateClassicCamera()
 	local module = RootCameraCreator()
 	
-	local tweenAcceleration = math.rad(220)
-	local tweenSpeed = math.rad(0)
-	local tweenMaxSpeed = math.rad(250)
+	local tweenAcceleration    = math.rad(220)
```

Comment:
No extra spaces needed here before the equal signs. Lets stay consistent with the existing style of the file.